time/src/test/java/net/jqwik/time/api/LocalTimeTests.java

```diff
@@ -352,6 +482,7 @@ void defaultShrinking(@ForAll Random random) {
 			assertThat(value).isEqualTo(LocalTime.of(0, 0, 0, 0));
 		}
 
+		//TODO: Too much time?
 		@Property
 		void shrinksToSmallestFailingValue(@ForAll Random random) {
```

Comment:
This test case works, but need 44 seconds on my machine. What should I do? Constrain the precision to Millis/Micros/Seconds?

Reply:
Reducing precision is an option. I also wonder whatthe default precision should be. I tend to seconds mit maybe millis is more what someone would expect?

Reply:
That's an idea. At the moment default precision is Nanos